l3kernel/testfiles/m3bitset001.lvt

```diff
@@ -0,0 +1,241 @@
+% !Mode:: "TeX:UTF-8:Main"
+\input{regression-test}
+\RequirePackage[enable-debug,check-declarations]{expl3}
+\documentclass{article}
+
+
+
+\begin{document}
+\ExplSyntaxOn
+\START\showoutput
```

Comment:
Using `\showoutput` here is a quick way to make a test, but almost all of the `expl3` tests use the `\TEST { }` or `\TESTEXP { } { }` set up with deliberate `\<thing>_show:N` or similar. That helps with programming stuff as we don't get the engine-dependent output that using typeset output can lead to.

I think before we merge, the tests need to be tidied up to be more similar to those in the other 'variable' modules.